SU2_CFD/src/solvers/CNSSolver.cpp

```diff
@@ -289,6 +330,9 @@ unsigned long CNSSolver::SetPrimitive_Variables(CSolver **solver_container, CCon
       }
     }
 
+    if (turb_model == NONE && SGSModelUsed)
```

Comment:
I don't think you need the extra bool for SGSModelUsed, if you just use ```if ( SGSModel )``` instead and it is not equal to null